aea/aea.py

```diff
@@ -20,7 +20,6 @@
 """This module contains the implementation of an Autonomous Economic Agent."""
 import logging
 from asyncio import AbstractEventLoop
-from concurrent.futures import Executor
```

Comment:
we are not using Executor for task manager anymore - `multiprocessing.Pool` gives much more control, especially on cancellation of workers.